diff --git a/node_modules/codeceptjs-resemblehelper/index.js b/node_modules/codeceptjs-resemblehelper/index.js
index 89da5dd..f0c1f15 100644
--- a/node_modules/codeceptjs-resemblehelper/index.js
+++ b/node_modules/codeceptjs-resemblehelper/index.js
@@ -6,6 +6,7 @@ const getDirName = require('path').dirname;
 const AWS = require('aws-sdk');
 const path = require('path');
 const sizeOf = require('image-size');
+const Container = require('codeceptjs/lib/container');
 
 /**
  * Resemble.js helper class for CodeceptJS, this allows screen comparison
@@ -29,6 +30,23 @@ class ResembleHelper extends Helper {
     return folderPath;
   }
 
+  resolveRelativePath(folderPath) {
+    let absolutePathOfImage = folderPath;
+    if (!path.isAbsolute(absolutePathOfImage)) {
+      absolutePathOfImage = path.resolve(global.codecept_dir, absolutePathOfImage) + "/";
+    }
+    let absolutePathOfReportFolder = global.output_dir;
+    // support mocha
+    if (Container.mocha() && Container.mocha().options.reporterOptions.reportDir) {
+      absolutePathOfReportFolder = Container.mocha().options.reporterOptions.reportDir;
+    }
+    // support mocha-multi-reporters
+    if (Container.mocha() && Container.mocha().options.reporterOptions.mochawesomeReporterOptions.reportDir) {
+      absolutePathOfReportFolder = Container.mocha().options.reporterOptions.mochawesomeReporterOptions.reportDir;
+    }
+    return path.relative(absolutePathOfReportFolder,absolutePathOfImage);
+  }
+
   /**
    * Compare Images
    *
@@ -170,11 +188,11 @@ class ResembleHelper extends Helper {
 
     if (mocha !== undefined && misMatch >= options.tolerance) {
       await mocha.addMochawesomeContext("Base Image");
-      await mocha.addMochawesomeContext(this._getBaseImagePath(baseImage, options));
+      await mocha.addMochawesomeContext(this.resolveRelativePath(this._getBaseImagePath(baseImage, options)));
       await mocha.addMochawesomeContext("ScreenShot Image");
-      await mocha.addMochawesomeContext(this._getActualImagePath(baseImage));
+      await mocha.addMochawesomeContext(this.resolveRelativePath(this._getActualImagePath(baseImage)));
       await mocha.addMochawesomeContext("Diff Image");
-      await mocha.addMochawesomeContext(this._getDiffImagePath(baseImage));
+      await mocha.addMochawesomeContext(this.resolveRelativePath(this._getDiffImagePath(baseImage)));
     }
   }
 
